### PR TITLE
add more usable ammo to syrigne gun

### PR DESCRIPTION
**Pull request by jorgun**
## About the PR

added cyostasis bluspace and darts to be used by the syringe gun

## Why / Balance

im not sure if it was an oversight or intentional.
why a cryostasis syringe filled whit 5 units of clorine and 15 units of fluoride heated up and loaded into on of these guns.  the first shoot will do about 170 damage to an unarmed target, the secound wil rip away any limb.
the third shoot will remove the floor and remove the head of the victim. 

## Requirements

- [x] I have read and am following the [Pull Request and Changelog Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html).
- [x] I have added media to this PR or it does not require an ingame showcase.

## Breaking changes

**Changelog**

:cl:
- tweak: cyostasissyring and more can now be used in the syringe guns.

### CODE REVIEW

**Comment by Aviu00** [comment]
That's too insane.

**Comment by gluesniffler** [comment]
Even as the person who preaches ignoring balance, 100u of any chemical in one shot is just stupidly broken LMAO

**Comment by a-person5660** [comment]
yea, maybe exclude the bluespace syringe from this lol.

**Comment by a-person5660** [comment]
also spelling error in your CL :trollface:, it should be;

> tweak: cyostasis syringe and more can now be used in syringe guns.

**Comment by gluesniffler** [comment]
> also spelling error in your CL :trollface:, it should be;
> 
> > tweak: cyostasis syringe and more can now be used in syringe guns.

aint no way you tried correcting them and also wrote it incorrectly 😭

**Comment by a-person5660** [comment]
> > also spelling error in your CL :trollface:, it should be;
> > > tweak: cyostasis syringe and more can now be used in syringe guns.
> 
> aint no way you tried correcting them and also wrote it incorrectly 😭

im dyslexic ok 😭

**Comment by Aineias1** [comment]
> > > also spelling error in your CL :trollface:, it should be;
> > > > tweak: cyostasis syringe and more can now be used in syringe guns.
> > 
> > aint no way you tried correcting them and also wrote it incorrectly 😭
> 
> im dyslexic ok 😭 

Cinema

**Comment by draftsmans** [comment]
the Syringe Gun is already used as a valdihunting tool most of the time. 100u of any dangerous chem is easy permakill. This will fuck over Lings and Heretics SO FUCKING HARD.

File: Resources/Prototypes/Entities/Objects/Fun/darts.yml [hunk 1]
```diff
@@ -85,6 +85,9 @@
       - !type:DoActsBehavior
         acts: [ "Destruction" ]
   - type: BalloonPopper
+  - type: Tag  # Goobstation
+    tags:
+    - SyringeGunAmmo
```
[the remark this answers is not in the document]
Comment (jorgun) [reply Resources/Prototypes/Entities/Objects/Fun/darts.yml:90]: coderabit that is not how tag syntaxt is

**Comment by Piras314** [comment]
darts are one thing, cryostasis and bluespace are very much another lmao

**Comment by Aidenkrz** [comment]
nuh uh

**Pull request closed without merging**